spec/config/data.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { NODES, NodeConfig } from '../../common/config/data';
+import { RPCNode } from '../../common/libs/nodes';
+import { Validator } from 'jsonschema';
+import { schema } from '../../common/libs/validators';
+import 'url-search-params-polyfill';
```

Comment:
unclear why we need this

Reply:
@dternyak it injects search params so that the Etherscan Nodes work as seen here:
```
 public encodeRequest(request: EtherscanRequest): string {
    const encoded = new URLSearchParams();
    Object.keys(request).forEach(key => {
      if (request[key]) {
        encoded.set(key, request[key]);
      }
    });
    return encoded.toString();
  }
```